README.md

```diff
@@ -1,22 +1,45 @@
-﻿# Project Title
+﻿# Chinook.DynamicMvvm
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-{Project tag line}
+The `Chinook.DynamicMvvm` library assists in MVVM (Model - View - ViewModel) development.
 
-{Small description of the purpose of the project}
+There are 3 main components to `Chinook.DynamicMvvm`:
+
+* [IViewModel](src/DynamicMvvm.Abstractions/ViewModel/IViewModel.md): a store of `IDisposable` that can be bound to a view and is meant to contain a set of `IDynamicProperties` and `IDynamicCommands`.
```

Comment:
Instead of "a store of IDisposable", I think we should focus on something that has properties and commands and implements `INotifyPropertyChanged` because VMs are mainly used for bindings.